.ci/kitchen-macosxmojave-py3

```diff
@@ -9,11 +9,12 @@ runTestSuite(
     golden_images_branch: 'master',
     jenkins_slave_label: 'kitchen-slave-mac',
     kitchen_platforms_file: '/var/jenkins/workspace/pre-golden-nox-platforms.yml',
-    nox_env_name: 'runtests-zeromq',
+    nox_env_name: 'pytest-zeromq',
     nox_passthrough_opts: '--ssh-tests',
     python_version: 'py3',
-    run_full: params.runFull,
+    //splits: ['unit', 'integration', 'multimaster'],
```

Comment:
Why are these in here commented out?

Reply:
Because not all of them are split and I don't want to test and try again to get that right, when/if we split

Reply:
Would it be useful to add that comment into the text here (and elsewhere we have the splits)?

Reply:
Removed. Git history will reveal them back